packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -560,6 +560,120 @@ function AsyncSortTable() {
 }
 ```
 
+## Column Resizing <VersionBadge version="beta" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+TableView supports resizable columns, allowing users to dynamically adjust the width of a column. To designate that a Column is resizable, provide it with the `allowsResizing` prop. This will render a draggable
+resizer handle that becomes visible on hover. Keyboard, touch, and screen reader users can start resizing by interacting with the target column's header and selecting the "Resize column" option
+from the dropdown.
+
+### Width values
```

Comment:
I guess the width configuration isn't really specific to column resizing. Wonder if people will miss it because it is a sub-heading?

Reply:
I'll keep this section here and restore the old width values section we had down in visual options then

Reply:
Not sure, it was more of a question for everyone

Reply:
I'd worry about people missing how to define widths if it was under resizing only. Can we invert it and have resizing be a sub header?

Reply:
IMO, it would be good to preserve the old section and keep the resizing as the main header. I feel like most people who are looking for column resizing will look specifically for "Resizing" as a header and keeping the old section as will also keep the page layout familiar to old users (no confusion as to if that visual option has disappeared and old links will still work)

Reply:
I agree, I would like width to be it's own heading/sub-heading and it could be referenced from the resizing section that seems ok